eng/common/scripts/modules/Package-Properties.psm1

```diff
@@ -0,0 +1,177 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules
+class PackageProps
+{
+    [string]$pkgName
+    [string]$pkgVersion
+    [string]$pkgDirectoryPath
+    [string]$pkgReadMePath
+    [string]$pkgChangeLogPath
+
+    PackageProps()
+    {
+    }
+    PackageProps(
+        [string]$pkgName,
+        [string]$pkgVersion,
+        [string]$pkgDirectoryPath
+    )
+    {
+        $this.pkgName = $pkgName
+        $this.pkgVersion = $pkgVersion
+        $this.pkgDirectoryPath = $pkgDirectoryPath
+
+        if (Test-Path (Join-Path $pkgDirectoryPath "README.md"))
+        {
+            $this.pkgReadMePath = Join-Path $pkgDirectoryPath "README.md"
+        } else
+        {
+            $this.pkgReadMePath = $null
+        }
+
+        if (Test-Path (Join-Path $pkgDirectoryPath "CHANGELOG.md"))
+        {
+            $this.pkgChangeLogPath = Join-Path $pkgDirectoryPath "CHANGELOG.md"
+        } else
+        {
+            $this.pkgChangeLogPath = $null
+        }
+    }
+}
+
+function Extract-PkgProps ($pkgPath, $pkgName)
+{
+    if ($Language -eq "net")
+    { Extract-DotNetPkgProps -pkgPath $pkgPath -pkgName $pkgName 
```

Comment:
Similar comment here about braces. I'd put the opening brace at the end of the line with the if statement on it.